README.md

```diff
@@ -1,3 +1,7 @@
+# DEPRECATED
+
+This branch deprecated in favor https://github.com/aws-samples/amazon-sumerian-hosts/tree/mainline2.0, please see https://github.com/aws-samples/amazon-sumerian-hosts/tree/mainline2.0#readme for more details.
+

```

Comment:
Do we need to update this link after renaming `mainline2.0 to mainline` in the future?

Reply:
I don't think we're going to rename. This link will work even if we change the default branch in repo